packages/core/src/components/carousel/stories/carousel.core.stories.tsx

```diff
@@ -1,141 +1,95 @@
-    <atom-carousel-item><div class="${itemClass}">Slide 5</div></atom-carousel-item>
-    <atom-carousel-item><div class="${itemClass}">Slide 6</div></atom-carousel-item>
-    <atom-carousel-item><div class="${itemClass}">Slide 7</div></atom-carousel-item>
-    <atom-carousel-item><div class="${itemClass}">Slide 8</div></atom-carousel-item>
-    <atom-carousel-item lazy="true">
-      <div class="${itemClass}">Slide 9
-        <img loading="lazy" width="100px" src="https://user-images.githubusercontent.com/3603793/257943112-fb180815-7bd7-45f7-ad14-bd1677079931.png"/>
-      </div>
-    </atom-carousel-item>
-  </atom-carousel>
-  <script>
-  const  atomCarousel${carouselStoryId} = document.querySelector('atom-carousel#carousel-${carouselStoryId}');
-    atomCarousel${carouselStoryId}.addEventListener('atomChange',(event)=>{
-  console.log('atomChange', event)
-    })
-    atomCarousel${carouselStoryId}.addEventListener('atomClickPrev',(event)=>{
-      console.log('atomClickPrev', event)
-
-    })
-    atomCarousel${carouselStoryId}.addEventListener('atomClickNext',(event)=>{
-      console.log('atomClickNext', event)
 
-    })
-  </script>
+const createCarousel = (args) => {
+  return html`
+    <atom-carousel
+      loop=${args.loop}
+      autoplay=${args.autoplay}
+      has-navigation=${args.hasNavigation}
+      has-pagination=${args.hasPagination}
+    >
+      <atom-carousel-item>
+        <div
+          style="background-color: var(--color-brand-primary-regular); color: #fff; height: 300px; display: flex; align-items: center; justify-content: center"
+        >
+          Example 1
+        </div>
+      </atom-carousel-item>
+      <atom-carousel-item>
+        <div
+          style="background-color: var(--color-brand-secondary-regular); color: #fff; height: 300px; display: flex; align-items: center; justify-content: center"
+        >
+          Example 2
+        </div>
+      </atom-carousel-item>
+      <atom-carousel-item>
+        <div
+          style="background-color: var(--color-contextual-success-regular); color: #fff; height: 300px; display: flex; align-items: center; justify-content: center"
+        >
+          Example 3
+        </div>
+      </atom-carousel-item>
+    </atom-carousel>
   `
 }
 
 export const Default: StoryObj = {
-  render: () =>
-    createComponent(
-      {
-        pagination: true,
-        spaceBetween: 40,
-        paginationClickable: true,
-        centeredSlides: true,
-      },
-      'item-default'
-    ),
+  render: (args) => createCarousel(args),
+  args: {
+    ...CarouselComponentArgs,
+  },
 }
 
-export const DisabledPagination: StoryObj = {
-  render: () =>
-    createComponent(
-      { pagination: false, slidesPerView: 3, spaceBetween: 40 },
-      'item-default'
-    ),
+export const Loop: StoryObj = {
+  render: (args) => createCarousel(args),
+  args: {
+    ...CarouselComponentArgs,
+    loop: true,
```

Comment:
Doubt: I didn't understand what this prop does. I didn't see any difference in its behavior